### DISCUSSION TITLE
The Cloud and the Climate: Navigating AI-Powered Futures

### DISCUSSION THREAD

**Discussion by SusDigiHumLab** [Green-Software-Foundation/discussions#176] [News]
Hi all,

Our report on the current state of sustainable cloud / sustainable AI / AI for sustainability is available as an exposure draft here:
https://docs.google.com/document/d/1O5PmODLJHlX5h0CGOnI4CGNgfK4atJIvqtJUCFaJrfc/edit?usp=sharing

All comments and feedback welcome.

All the best,
Jo

**Comment by franciscrot** [comment]
And it's here! The final* report. GreenOps, sustainable cloud, energy procurement, carbon offsetting, AI for sustainability and sustainable AI, and more. Hope that folks here find it of some use:
https://sussexhumanitieslab.wordpress.com/2024/09/29/the-cloud-and-the-climate-navigating-ai-powered-futures/

(*) Well probably not ENTIRELY final (you know how it is) so please feel free to send your feedback, comments, corrections and queries in.
